generator/templates/go/client.go.mustache

```diff
@@ -74,23 +77,23 @@ func (c *Client) token(ctx context.Context) (string, error) {
 {{#DocLines}}
 // {{{.}}}
 {{/DocLines}}
-type {{NameToCamel}} struct {
+type {{Name}} struct {
     client *Client
     baseURL   string
 }
 
 {{#DocLines}}
 // {{{.}}}
 {{/DocLines}}
-func (c *Client) {{NameToCamel}}() *{{NameToCamel}}{
-    return &{{NameToCamel}}{client: c, baseURL: "https://{{DefaultHost}}"}
+func (c *Client) {{Name}}() *{{Name}}{
+    return &{{Name}}{client: c, baseURL: "https://{{DefaultHost}}"}
 }
 
 {{#Methods}}
 {{#DocLines}}
 // {{{.}}}
 {{/DocLines}}
-func (s *{{ServiceNameToCamel}}) {{NameToCamel}}(ctx context.Context, req *{{InputTypeName}}) (*{{OutputTypeName}}, error) {
+func (s *{{Name}}) {{NameToCamel}}(ctx context.Context, req *{{InputTypeName}}) (*{{OutputTypeName}}, error) {
```

Comment:
Hmm... maybe `Name` is slightly confusing here, should that be `ServiceName` so we don't confuse it with the name of the method?